### PR TITLE
{cae,vis}[GCCcore/12.3.0] occt v7.8.0, FreeImage v3.18.0

### PR DESCRIPTION
**Pull request by furstj**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/o/occt/occt-7.8.0-foss-2023a.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+name = 'occt'
+version = '7.8.0'
+
+homepage = 'https://www.opencascade.com/'
+description = """Open CASCADE Technology (OCCT) is an object-oriented C++
+class library designed for rapid production of sophisticated domain-specific
+CAD/CAM/CAE applications."""
+
+toolchain = {'name': 'foss', 'version': '2023a'}
```
Comment (Micket) [inline easybuild/easyconfigs/o/occt/occt-7.8.0-foss-2023a.eb:11]: I see no indications that this uses either BLAS or MPI, we can definitely move this. In fact, i'm pretty sure it's mostly just header only so I say lets try moving it to GCCcore.

**Review by Micket — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/o/occt/occt-7.8.0-foss-2023a.eb (hunk 1 above)
Comment (furstj) [reply easybuild/easyconfigs/o/occt/occt-7.8.0-foss-2023a.eb:11]: You are right. The toolchain is changed to GCCcore in my last commit

Comment (Micket) [reply easybuild/easyconfigs/o/occt/occt-7.8.0-foss-2023a.eb:11]: your PR still contains the old version, clear it out with `--update-pr 19864 :occt-7.8.0-foss-2023a.eb`

File: easybuild/easyconfigs/o/occt/occt-7.8.0-GCCcore-12.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+toolchain = {'name': 'GCCcore', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/Open-Cascade-SAS/OCCT/archive/refs/tags']
+sources = ['V7_8_0.tar.gz']
+checksums = ['096cd0f268fa9f6a50818e1d628ac92ecf87e10fd72187e2e8d6be57dfe12530']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('Doxygen', '1.9.7'),
```
Comment (Micket) [inline easybuild/easyconfigs/o/occt/occt-7.8.0-GCCcore-12.3.0.eb:19]: needs binutils

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 3 out of 3 (2 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/9a30c73b233d9627a4f91a7a14712f21 for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
@boegelbot please test @ jsc-zen3

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-skylake-build - Linux Rocky Linux 8.9, x86_64, Intel Xeon Processor (Skylake, IBRS, no TSX), Python 3.6.8
See https://gist.github.com/Micket/61b8b24ed8217514a79e0a68e8752ea9 for a full test report.

**Comment by Micket** [comment]
Going in, thanks @furstj!

**Pull request merged**